src/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.Unix.BackingObject_File.cs

```diff
@@ -37,6 +38,10 @@ private static FileStream CreateSharedBackingObject(Interop.libc.MemoryMappedPro
         // ---- PAL layer ends here ----
         // -----------------------------
 
-        private static readonly string TmpPathPrefix = Path.Combine(Path.GetTempPath(), MemoryMapObjectFilePrefix);
+        private static readonly string s_tempMapsDirectory = Path.Combine(
+            Path.GetTempPath(),
```

Comment:
Right now PersistedFiles only uses Path.Combine; though I do understand the argument about Pipes not being able to use Directory.CreateDirectory (therefore not wanting to go reparse the path to do the equivalent to `mkdir -p`).

This call could use the helper method, as witnessed by the single Path.Combine here.
Pipes could accomplish it by either exposing something like `CreateTempFeatureDirectoryRecursive(Action<string> mkdir)`; or exposing `GetTempTopLevelPath()` and `GetTempProductPath()`.  Or MemoryMappedFiles using the helper, and Pipes having a comment that it's really `PersistedFiles.GetTempFeaturePath`, but written longhand because it can't use Directory.CreateDirectory.

Certainly not a "must" comment; just trying to avoid a followup change where we change this to use a helper if we see it coming in the short-to-mid-term.